pkg/instrumentation/upgrade/upgrade.go

```diff
@@ -52,6 +58,28 @@ type InstrumentationUpgrade struct {
 	DefaultAutoInstApacheHttpd string
 	DefaultAutoInstNginx       string
 	DefaultAutoInstGo          string
+	defaultAnnotationToConfig  map[string]autoInstConfig
+}
+
+func NewInstrumentationUpgrade(client client.Client, recorder record.EventRecorder, cfg config.Config) *InstrumentationUpgrade {
+	defaultAnnotationToConfig := map[string]autoInstConfig{
+		constants.AnnotationDefaultAutoInstrumentationApacheHttpd: autoInstConfig{constants.ApacheHttpd, cfg.EnableApacheHTTPAutoInstrumentation()},
```

Comment:
@TylerHelmuth `constants.ApacheHttpd` is used here for mapping corresponding annotation, it previously use `featuregate.EnableApacheHttpAutoInstrumentationSupport.ID()` for logging when a featuregate is disabled, I then follow the same way how `annotations` are mapped to `featuregate`.

Reply:
I think we should be logging the full command line flag (`enable-apache-http-instrumentation`) like the feature gate did.  `apache-httpd` is not the name of the flag